gammacat/webpage.py

```diff
@@ -62,11 +62,25 @@ def make_references_data(self):
         references_data = []
         for reference_id in reference_ids:
             ads_url = f'https://ui.adsabs.harvard.edu/#abs/{reference_id}'
+            year = {reference_id[:4]}
+            in_folder = f'{gammacat_info.input_url}/data/{year}/{urllib.parse.quote(reference_id)}'
+            out_folder = f'{gammacat_info.output_url}/data/{year}/{urllib.parse.quote(reference_id)}'
```

Comment:
I think that 'gammacat_info.base_dir/input'  works instead of 'gammacat_info.input_url' as well (analogeously for output) . But I am not sure whether RTD handles this right.

Please test it on RTD and if it turns out that base_dir works then the changes in gammacat/info.py are not necessary (see my comment there).